TestFoundation/TestNotificationQueue.swift

```diff
@@ -117,7 +117,7 @@ FIXME SR-4280 timeouts in TestNSNotificationQueue tests
             // post 2 notifications for the NSDefaultRunLoopMode mode
```

Comment:
> NSDefaultRunLoopMode

This should be `RunLoopMode.defaultRunLoopMode`.

https://developer.apple.com/documentation/foundation/runloopmode/1409732-defaultrunloopmode